### DISCUSSION TITLE
curious about the  key word training system

### DISCUSSION THREAD

**Discussion by 343333333** [deepmodeling/deepmd-kit#969] [Q&A]
the document say , if there have four set which is set.01 set.02 set.03 set.04. the first three sets will be training set and the last one would be test set.
what if i have four system,which is:
```
    "training": {
        "systems": [
            "../data.iters/iter.000000/02.fp/data.000"
            "../data.iters/iter.000001/02.fp/data.001",
            "../data.iters/iter.000002/02.fp/data.002"
            "../data.iters/iter.000003/02.fp/data.003"
        ],
```
and every data.00x have just one set named set.000. so, under this circumstance , is there only one set in the whole system that does the training set and the test set or is there four sets separately(which means that the set.000 in the data.003 would be the test set in dpmd-kit's training)

**Comment by njzjz** [comment]
See #217.

**Reply by 343333333** [reply]
thx for replying.
So, the four system's set.000 will purely be used for training and testing?

**Reply by njzjz** [reply]
Right.

**Reply by 343333333** [reply]
thanks.
